app-backend/database/src/main/scala/com/azavea/rf/database/tables/ScenesToProjects.scala

```diff
@@ -1,26 +1,25 @@
 package com.azavea.rf.database.tables
 
-import com.azavea.rf.database.fields._
-import com.azavea.rf.database.query._
-import com.azavea.rf.database.sort._
+import java.util.UUID
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+
 import com.azavea.rf.database.{Database => DB}
 import com.azavea.rf.database.ExtendedPostgresDriver.api._
 import com.azavea.rf.datamodel._
-
-import com.typesafe.scalalogging.LazyLogging
 import com.lonelyplanet.akka.http.extensions.PageRequest
-import spray.json._
+import com.typesafe.scalalogging.LazyLogging
```

Comment:
It seems most (if not all) companions objects of our Table definitions extend `LazyLogging`, so I'm going to leave it.